### PR TITLE
SemesterCourse Controller Get All endpoint

### PR DESCRIPTION
**Pull request by JaAfghan**
Semester Course Controller Get All endpoint Completed

### CODE REVIEW

File: OtripleS.Web.Api/Controllers/SemesterCourseController.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
```
Comment (syedib) [inline OtripleS.Web.Api/Controllers/SemesterCourseController.cs]: **CopyRights** text needs to be inserted

Comment (JaAfghan) [reply OtripleS.Web.Api/Controllers/SemesterCourseController.cs]: already inserted in second Commit 

Copyright notice added in SemseterCourse Controller

File: OtripleS.Web.Api/Controllers/SemesterCourseController.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        public SemesterCourseController(ISemesterCourseService semesterCourseService)=>
+            this.semesterCourseService = semesterCourseService;
+
+        [HttpGet]
+        public ActionResult<IQueryable<SemesterCourse>> GetAllSemesterCourse()
+        {
+            try
+            {
+                IQueryable storageSemesterCourse = this.semesterCourseService.RetrieveAllSemesterCourses();
+                return Ok(storageSemesterCourse);
```
Comment (syedib) [inline OtripleS.Web.Api/Controllers/SemesterCourseController.cs:33]: There should be a new line between **IQueryable storageSemesterCourse** and **return Ok(storageSemesterCourse)**

File: OtripleS.Web.Api/Controllers/SemesterCourseController.cs (hunk 1 above)
Comment (syedib) [reply OtripleS.Web.Api/Controllers/SemesterCourseController.cs]: Ok Thanks

**Comment by hassanhabib** [comment]
@JaAfghan looks like you got some conflicts.

**Pull request closed without merging**